Doppler.Currency/SlackHookServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Net.Http;
+using CrossCutting;
+using CrossCutting.SlackHooksService;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Doppler.Currency
+{
+    public static class SlackHookServiceCollectionExtensions
+    {
+        public static IServiceCollection AddSlackHook(this IServiceCollection services)
+        {
+            services.AddSingleton<ISlackHooksService>(provider =>
+            {
+                var configuration = provider.GetService<IConfiguration>();
+                var section = configuration.GetSection("SlackHook");
+
+                if (!section.Exists()) 
```

Comment:
What do you think about check if the `Uri` value is null or empty instead of check if the section exist?
And leve the `appsettings.json` with empty settings.
Maybe use a Service Factory class could be a good alternative.